File: micro-digiservices/ecreditservice/src/main/java/io/digiservices/ecreditservice/dto/FicheSignaletiqueResponse.java
package io.digiservices.ecreditservice.dto;

import lombok.Data;

import java.util.Map;

@Data
public class FicheSignaletiqueResponse {
    private String status;
    private Integer code;
    private Map<String, Object> data;
    private Map<String, Object> metadata;
    private String errorCode;
    private String message;
    private String codCliente;
    private Long timestamp;
}

File: frontend/ultima-ng-19.0.0/src/app/interface/FicheSignaletiqueResponse.ts
import { FicheSignaletiqueMetadata, FicheSignaletiqueWithSolde } from './FicheSignaletiqueWithSolde';

export interface FicheSignaletiqueResponse {
    time: number;
    timeStamp: number;
    code: number;
    statusCode: number;
    path: string;
    httpStatus: string;
    status: string;
    message: string;
    data: {
        metadata: FicheSignaletiqueMetadata;
        code: number;
        data: FicheSignaletiqueWithSolde;
        status: string;
    };
}
